app/poros/async_request/json_web_token.rb

```diff
@@ -0,0 +1,16 @@
+require 'jwt'
+
+module AsyncRequest
+  class JsonWebToken
+    def self.encode(job_id, expiration = Time.zone.now + 1.day)
```

Comment:
maybe config for expiration?

Reply:
Ok, I need to check how do we set some config for the gem